netdata/netdata
# prefer cap over setuid for sysetmd-journal in installer

##### Summary

Could be a part of #15733 but that PR seems to be ready to merge.

##### Test Plan

- check `sysetmd-journal.plugin` permissions and capabilities and
- check if `Systemd-journal` function works

##### Additional Information


<details> <summary>For users: How does this change affect me?</summary>
  
</details>
